protocol/getrelayblockslistener/getrelayblockslistener.go

```diff
@@ -0,0 +1,42 @@
+package getrelayblockslistener
```

Comment:
Please rename this to either getrelayblocks or relayblockslistener or something similar.